src/ViewModels/src/TreeViewItems/CfInstanceViewModel.cs

```diff
@@ -12,12 +14,14 @@ public class CfInstanceViewModel : TreeViewItemViewModel
         internal static readonly string _emptyOrgsPlaceholderMsg = "No orgs";
         internal static readonly string _loadingMsg = "Loading orgs...";
         internal static readonly string _getOrgsFailureMsg = "Unable to load orgs";
+        private static IErrorDialog _dialogService;
```

Comment:
beginning to see a pattern :)

Reply:
saving this and all the above changes for a new issue that will cover changing the use of the `static` keyword in all files, where necessary

Reply:
@jkonicki I don't mind doing all the other fixes in this ticket as well if that is what you would like. I just didn't know if for the sake of keeping things organized, we should split it up.

Reply:
but I will go ahead and make the change that chris mentioned here

Reply:
Update: I went ahead and started the process of getting rid of the `static` modifier in all files across al projects. so far, it has been fine, but if changing the keywords does cause any code related issues, I will take note of it and see what other options we have.

Reply:
Update #2: After talking with Chris, it was determined that _some_ of the Singletons that are being instantiated in the project should actually keep the `static` keyword( i.e. the `Commands` files in the VS project). Therefore, I am shifting back to only focusing on removing the `static` keyword from `Service` instances.